certif/translations/en.json

```
@@ -590,7 +590,9 @@
           "summary": "Récapitulatif"
         },
         "candidates-list": {
-          "import-success": "The candidate' list has been successfully uploaded."
+          "import-fail-prefix": "No candidate has been uploaded.<br/>",
```

Pourquoi mettre le `<br>` dans la trad et pas dans le template (cela permettrait par ailleurs de retirer le `{ htmlSafe: true }`)
```suggestion
          "import-fail-prefix": "No candidate has been uploaded.",
```

---

Il n'y a pas de template, c'est le toaster du service de notification qui est utilisé ici